expan/core/statistical_test.py

```diff
@@ -33,17 +33,18 @@ class DerivedKPI(KPI):
     
     :param name: name of the kpi
     :type  name: str
-    :param formula: formula of the kpi. It should be a ratio of two column. e.g. 'revenue/session'
```

Comment:
maybe we can also explain in the docstring of the class that a derived kpi is a ratio of two column, which is passed as numerator and denominator.